app/src/main/java/com/concordium/wallet/core/arch/BackendObserver.kt

```diff
@@ -11,7 +11,7 @@ import com.concordium.wallet.core.backend.BackendResource
 
 class BackendObserver<T> : Observer<BackendResource<T>> {
 
-    override fun onChanged(backendResource: BackendResource<T>?) {
+    override fun onChanged(backendResource: BackendResource<T>) {
         onDone()
         if (backendResource != null) {
```

Comment:
Redundant check as the var isn't nullable anymore.

Reply:
fixed